llvm/lib/Target/AIE/AIEPostSelectOptimize.cpp

```diff
@@ -438,6 +446,120 @@ bool duplicateAdressingRegs(MachineBasicBlock &MBB, MachineRegisterInfo &MRI) {
   return tryToDuplicateLoadUse(LoadUses, NonLoadUses, MRI, TII);
 }
 
+bool getGlobalValue(const MachineInstr *MI,
+                    SmallPtrSet<const Value *, 4> &GVSet,
+                    MachineRegisterInfo &MRI) {
+
+  if (MI->isCopy())
+    MI = getDefIgnoringCopiesAndBitcasts(MI->getOperand(1).getReg(), MRI);
```

Comment:
Nit: Why not unconditionally call `MI = getDefIgnoringCopiesAndBitcasts(MI->getOperand(1).getReg(), MRI);`?